cloud-data-protection/src/services/demoService.ts

```diff
@@ -0,0 +1,59 @@
+import {ServiceBase} from "services/base/serviceBase";
+import {AxiosResponse, CancelToken} from "axios";
+import {http} from "common/http";
+import FileUploadResult from "services/result/demo/fileUploadResult";
+import FileInfoResult from "services/result/demo/fileInfoResult";
+import {FileDownloadResult} from "services/result/demo/fileDownloadResult";
+import {readFileName} from "common/parser/contentDisposition";
+import fileDownload from "js-file-download";
+
+class DemoService extends ServiceBase {
+    public static readonly maxFileSize: number = 25 * 1024 * 1024;
+
+    public async upload(file: File, cancelToken?: CancelToken): Promise<FileUploadResult> {
+        const formData = new FormData();
+
+        formData.append('File', file);
+
+        return await http.post('/demo/file', formData, { cancelToken: cancelToken, headers: { 'content-type': 'multipart/form-data' } })
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7431242)
```suggestion
        return await http.post('/demo/file', formData, { cancelToken, headers: { 'content-type': 'multipart/form-data' } })
```